client/signup/steps/woocommerce-install/select-site/style.scss

```diff
@@ -3,6 +3,16 @@
 @import '@wordpress/base-styles/_breakpoints.scss';
 @import '@wordpress/base-styles/_mixins.scss';
 
+body.is-section-signup {
+	.signup__step.is-select-site {
+		.is-wide-layout {
+			max-width: 1024px;
+			margin: 0 20px;
+			padding: 0 20px;
+		}
+	}
+}
```

Comment:
Do you recall why you added these? Removed them in c534b90b359ede340fbe847328bc2b0eb8d3e523 because all it seemed to do was force the whole selector left aligned. I'll this back if I'm misunderstanding something here.

Reply:
@retrofox ^